content/en/docs/Tracing/tempo-walkthrough.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Tracing with Grafana Tempo
+description: Tracing with Grafana Tempo.
+weight: 10
+---
+
+## Cluster Setup
+
+TODO
+
+```shell
+kind create cluster --config - <<EOF
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+    kubeadmConfigPatches:
+      - |-
+        kind: InitConfiguration
+        nodeRegistration:
+          kubeletExtraArgs:
+            node-labels: "ingress-ready=true"
+    extraPortMappings:
+      - containerPort: 80
+        hostPort: 80
+        protocol: TCP
+      - containerPort: 443
+        hostPort: 443
+        protocol: TCP
+  - role: worker
+  - role: worker
+  - role: worker
```

Comment:
Let's just ask one worker. 3 is quite a lot for this tutorial.

Reply:
Done